### ISSUE TITLE
XIVAPI backend

### ISSUE THREAD

**Issue by karashiiro** [xivapi/godestone#13]
**Is your feature request related to a problem? Please describe.**
Waiting on updates for bingode is not ideal.

**Describe the solution you'd like**
An XIVAPI backend, so that no library updates need to happen to keep software functional.

**Comment by karashiiro** [comment]
Created blank project at https://github.com/xivapi/godexivapi

**Comment by karashiiro** [comment]
Oh, I probably want to do this after #7 so that all future provider development can be done on a clean foundation.

**Comment by karashiiro** [comment]
Will remake this in this repo, under a pkg subdirectory.

**Comment by karashiiro** [comment]
Putting this on hold, I need a decent way of dealing with XIVAPI's rate limit, it's really not suited to massive numbers of lookups.
